src/Catalyst.Node.POA.CE.Tests/IntegrationTests/Config/GlobalConfigTests.cs

```diff
@@ -0,0 +1,117 @@
+
+namespace Catalyst.Node.POA.CE.Tests.IntegrationTests.Config
+{
+    public class GlobalConfigTests : FileSystemBasedTest
+    {
+        public static readonly List<object[]> Networks =
+            new List<NetworkType> {NetworkType.Devnet, NetworkType.Mainnet, NetworkType.Testnet}
+               .Select(n => new object[] {n}).ToList();
+
+        private IEnumerable<string> _configFilesUsed;
+        private ContainerProvider _containerProvider;
+
+        public GlobalConfigTests(ITestOutputHelper output) : base(output) { }
+
+        [Theory]
+        [MemberData(nameof(Networks))]
+        public void Registering_All_Configs_Should_Allow_Resolving_CatalystNode(NetworkType network)
+        {
+            _configFilesUsed = new[]
+                {
+                    Constants.NetworkConfigFile(network),
+                    Constants.SerilogJsonConfigFile
+                }
+               .Select(f => Path.Combine(Constants.ConfigSubFolder, f));
+
+            _containerProvider = new ContainerProvider(_configFilesUsed, FileSystem, Output);
+
+            SocketPortHelper.AlterConfigurationToGetUniquePort(_containerProvider.ConfigurationRoot);
+
+            _containerProvider.ConfigureContainerBuilder();
+
+            var containerBuilder = _containerProvider.ContainerBuilder;
+            containerBuilder.RegisterType<CatalystNodePoa>().As<ICatalystNode>();
+            containerBuilder.RegisterType<ConsoleUserOutput>().As<IUserOutput>();
+            containerBuilder.RegisterType<ConsoleUserInput>().As<IUserInput>();
+            containerBuilder.RegisterInstance(Substitute.For<IPeerDiscovery>()).As<IPeerDiscovery>();
+            containerBuilder.RegisterModule(new KeySignerModule());
+            containerBuilder.RegisterModule(new ConsensusModule());
+            containerBuilder.RegisterModule(new DfsModule());
+            containerBuilder.RegisterModule(new KvmModule());
+            containerBuilder.RegisterModule(new LedgerModule());
+            containerBuilder.RegisterModule(new RpcServerModule());
+            containerBuilder.RegisterModule(new MempoolModule());
+            containerBuilder.RegisterModule(new KeystoreModule());
+            containerBuilder.RegisterModule(new BulletProofsModule());
+            containerBuilder.RegisterModule(new AuthenticationModule());
+
+            containerBuilder.RegisterAssemblyTypes(typeof(CoreLibProvider).Assembly)
+               .AssignableTo<IMapperInitializer>().As<IMapperInitializer>();
+            containerBuilder.RegisterType<MapperProvider>().As<IMapperProvider>()
+               .SingleInstance();
+
+            using (var scope = _containerProvider.Container.BeginLifetimeScope(CurrentTestName + network))
+            {
+                _ = scope.Resolve<ICatalystNode>();
+            }
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+            if (!disposing) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)